imap_processing/idex/idex_packet_parser.py

```diff
@@ -0,0 +1,286 @@
+import logging
+
+import bitstring
+import numpy as np
+import xarray as xr
+from space_packet_parser import parser, xtcedef
+
+from imap_processing import packet_definition_directory
+
+TWENTY_MICROSECONDS = 20 * (10 ** (-6))
+
+
+class IDEXPacketParser:
+    """
```

Comment:
Follow numpydoc conventions according to the style guide: https://imap-processing.readthedocs.io/en/latest/development/style-guide/api-documentation.html

```python
Parameters
------------
````

There is also no filename parameter, and I think parameters can be documented within the constructor below and put attributes here.